recipes/pytspack/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pytspack" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/noaa-oar-arl/pytspack/{{ version }}.tar.gz
+  sha256: 3e35e2c56ace4452dc3aadf030f081581da7257044cd66f24c3343f396cdca1e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - python >=3.6
```

Comment:
I think ``python`` should not be here, it should be just in ``host`` and ``run``
```suggestion
```
What was your idea for adding it here?

Reply:
removed